airflow/models/variable.py

```diff
@@ -47,12 +47,14 @@ class Variable(Base, LoggingMixin):
     id = Column(Integer, primary_key=True)
     key = Column(String(ID_LEN), unique=True)
     _val = Column('val', Text)
+    description = Column(Text)
     is_encrypted = Column(Boolean, unique=False, default=False)
 
-    def __init__(self, key=None, val=None):
+    def __init__(self, key=None, val=None, description=None):
         super().__init__()
         self.key = key
         self.val = val
+        self.description = description
```

Comment:
Can we add some tests for this feature please

Reply:
@kaxil Since this is a feature only viewable/editable from the UI, what tests would you suggest? Just including the description field in [this line](https://github.com/apache/airflow/blob/master/tests/www/test_views.py#L275) would suffice? 
Thanks in advance!

Reply:
Probably just a separate test in that file, the simply tests that you can set description to a variable, which is peristed to db and when you do `Variable.get` you get the same description field

Reply:
@kaxil The thing is that following @kurtqq 's suggestion in [the issue](https://github.com/apache/airflow/issues/12413#issuecomment-809671348), I didn't modify the `Variable.get` method so it won't return the description. 

I may be missing something though, let me know what do you think.

Reply:
That is debatable :) -- but anyways for now you could just do `session.query(Variable.key, Variable.description).first()` and just verify that

Reply:
Thanks a lot for the tips! I rebased, implemented it and pushed in my branch but I guess a new PR is needed, right?

Reply:
Yup : )

Reply:
I've created #15400. Thanks again!